index.js

```diff
@@ -78,9 +80,16 @@ var run = function(options) {
         // JSON
         else {
           // some reports can be slimmed down for direct rendering
-          if (options.slim && report.slim) delete data.data;
-
-          writeReport(name, JSON.stringify(data, null, 2), ".json", done);
+          if (options.slim && report.slim) {
+            delete data.data;
+            writeReport(name, JSON.stringify(data, null, 2), ".json", done);
+          } else if (options["write-to-database"]) {
```

Comment:
This makes the use of `--slim` and `--write-to-database` exclusive, which I don't think is the intent. Even when data gets written to the database, we're still using the `--publish` flag, because I assume we still want to publish JSON files to the web. In that case, `--slim` and other options that affect JSON output need to be factored in.